src/app/pages/quickMaps/pages/baselineDetails/monthlyCard/monthlyCard.component.html

```diff
@@ -19,43 +19,43 @@
             <td mat-cell *matCellDef="let element">{{ element.unitPerc }}</td>
           </ng-container> -->
           <ng-container matColumnDef="vegetablesPerc">
-            <th mat-header-cell *matHeaderCellDef mat-sort-header>vegetablesPerc</th>
```

Comment:
It's good to have these.  They don't feed into the dialog though so maybe we could look at passing in column titles to the dialog table in the future too.